commerce-lang/src/main/resources/content/Language.properties

```diff
@@ -394,6 +394,7 @@ retry=Retry
 rounding-mode=Rounding Mode
 rounding-type=Rounding Type
 savings=Savings
+search-order=Search order…
```

Comment:
should have a capital `O` -- `Search Order...`. Language values should (almost) always be [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case) if a full sentence or [title case](https://en.wikipedia.org/wiki/Letter_case#Title_case) if not.

Reply:
Done it!